gateway-manager/src/manage_kong.py

```diff
@@ -280,7 +280,7 @@ def _realm_in_route(route):
     _remove_service_and_routes(name, routes_fn)
 
 
-def handle_app(action, name, options=None):
```

Comment:
I hadn't set the default value -> `{}` because the default is then mutable, but if we only read from it then it doesn't matter and this reads more clearly so :+1:

Reply:
I tried your changes in `aether-bootstrap` and I couldn't add any service :disappointed: